solid-common-lib/src/index.ts

```diff
@@ -11,65 +11,94 @@ export { url } from "./types";
  * @returns The URL's to the pods the user has access to
  */
 export async function listPodUrls(session: Session): Promise<url[]> {
-    const urls = await getPodUrlAll(session.info.webId!, {
-        fetch: session.fetch,
-    });
-    if (urls.length === 0) {
-        // TODO this is a guess and should be much more resilient
-        urls.push((new URL('../', session.info.webId!)).toString())
-    }
-    return urls;
+  const webId = session.info.webId!
+  const urls = await getPodUrlAll(webId, {
+    fetch: session.fetch,
+  });
+
+  const fetchedUrl = await fetchStorageTriple(webId);
```

Comment:
This call can throw an error, even though the original `getPodUrlAll` is resolving an url. 
An example to test this with is a webId from [inrupt pod spaces](https://start.inrupt.com/profile) ([ESS](https://docs.inrupt.com/ess/latest/)) or [solidcommunity.net ](https://solidcommunity.net/) ([NSS](https://github.com/nodeSolidServer/node-solid-server)).

Only webIDs from Community Solid Servers did not crash `fetchStorageTriple`.

Can you make it so that fetching storages from other solid servers do not crash?

I've tested it with following code:
```js
import { listPodUrls} from 'loama-common'

async function main() {
    // WebID self hosted CSS
    const session = { fetch: fetch, info: {webId: "https://pod.woutslabbinck.com/profile/card#me"}}
    const urls = await listPodUrls(session);
    console.log(urls)

    // WebID from inrupt solid server
    const sessionInrupt = { fetch: fetch, info: {webId: "https://id.inrupt.com/woutslabbinck"}}
    const urlsInrupt = await listPodUrls(sessionInrupt);
    console.log(urlsInrupt)

    // WebID from node solid server
    const sessionNss = { fetch: fetch, info: {webId: "https://woslabbi.solidcommunity.net/profile/card#me"}}
    const urlsNss = await listPodUrls(sessionNss);
    console.log(urlsNss)
}
main()
```

Reply:
Should be fixed with the latest commit